website/docs/r/sddc.html.markdown

```diff
@@ -201,6 +203,8 @@ In addition to arguments listed above, the following attributes are exported:
 
 * `cluster_info` - Information about cluster like id, name, state, host instance type.
 
+* `sddc_size` - Information about SDDC size like vc_size, nsx_size and SDDC size.
```

Comment:
this sentence is a little confusing with SDDC size being repeated

Reply:
Changed.